apps/shell/app.js

```diff
@@ -52,19 +56,39 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(process.env.PASSENGER_BASE_URI || '/', router);
 
 // Setup websocket server
-var server = new http.createServer(app);
-var wss = new WebSocket.Server({ noServer: true });
+const server = new http.createServer(app);
+const wss = new WebSocket.Server({ noServer: true });
+
+let whitelist = [];
```

Comment:
It's a little longer, but lets rename this from whitelist to host_whitelist. Elsewhere in OnDemand we deal with other whitelists (file paths, group acls, etc.) so its helpful when reading the code to know what we are whitelisting.